.github/workflows/merge-conflict-labeler.yaml

```diff
@@ -13,4 +13,4 @@ jobs:
       - uses: mschilde/auto-label-merge-conflicts@v2.0
```

Comment:
Don't use this one. It's no longer maintained and doesn't unlabel PRs when there are no conflicts (it checks for mergeable state, so if the PR isn't approved or doesn't go through all the branch protection rules, the labeling it's kept)

Use ``eps1lon/actions-label-merge-conflict@v2.0.1`` instead, which unlabels when there are truly no merge conflicts, skipping the rest of circunstancies (you can set it up to take care of them as well though).

Check Vue source: https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/merge-conflicts.yml

Reply:
Thanks - didn't know that. Will update it here and in android/sdk repo :)

Reply:
Updated, some differences with the vue version:
- Only run on our repo (a fork probably doesn't have a GH_TOKEN set and a fork doesn't need the action anyway)
- Run on all pushes - sometimes a PR doesn't target the master branch but it can still have conflicts